### PR TITLE
More integration tests

### PR DESCRIPTION
**Pull request by NonStatic2014**

### CODE REVIEW

File: onnxruntime/test/hosting/integration_tests/function_tests.py [hunk 1]
```diff
@@ -98,5 +102,175 @@ def test_mnist_invalid_url(self):
...
+
+        with open(input_data_file, 'r') as f:
+            request_payload = f.read()
+
+        r = requests.post(url, headers=request_headers, data=request_payload)
+        self.assertEqual(r.status_code, 400)
+        self.assertEqual(r.headers.get('Content-Type'), 'application/json')
+        self.assertTrue(r.headers.get('x-ms-request-id'))
+        self.assertEqual(r.headers.get('x-ms-client-request-id'), 'This~is~my~id')
+        self.assertEqual(r.content.decode('utf-8'), '{"error_code": 400, "error_message": "Missing or unknown \'Content-Type\' header field in the request"}\n')
```
Comment (tmccrmck) [inline onnxruntime/test/hosting/integration_tests/function_tests.py:124]: I'd rather not hard code error messages because they are subject to change

**Review by tmccrmck — APPROVED** [review]

File: onnxruntime/test/hosting/integration_tests/function_tests.py (hunk 1 above)
Comment (tmccrmck) [reply onnxruntime/test/hosting/integration_tests/function_tests.py:124]: But it's fine if you want for now

Comment (NonStatic2014) [reply onnxruntime/test/hosting/integration_tests/function_tests.py:124]: The integration test is from the customer/caller perspective.  If error message is changed, we need to update the test.

**Comment by NonStatic2014** [comment]
Stuck on the build. I will create a new one.

**Pull request closed without merging**